Serenity updates - October 2020 - Titles and structure (#783)
akeneo/pim-helpcenter: content/updates/2020-10/2-Rules-automatic-description-creation-and-new-efficient-filters.md

---
pim_announcement_img: ./img/.png
pim_announcement_alt_img: ""
pim_announcement_audience:
- EE
---

> # Rules: automatic description creation and new efficient filters

Reviewer: ```suggestion
# A new rules engine action and new filters for conditions
```

## A new action to automatize your description creation
::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
:::

> ## New filters in conditions

Reviewer: ```suggestion
## New filters for conditions
```
? not sure if it's "in" or "for"
